metadataserver/types/annotations.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+const (
+	IamRoleArnAnnotation                = "iam.amazonaws.com/role"
```

Comment:
I think it'd be nice to have a package with our k8s annotations across all projects. Maybe titus-api-definitions?

Reply:
Or even a single package amongst the executor.

Reply:
I think it should be a separate package - not the executor (we already have these names duplicated in the mutating webhook and the agent controllers) or api-definitions (I'd rather not have to vendor a bunch of generated proto files when all I want is a few golang constants).

Reply:
... doesn't have to happen as part of this PR, but we should do it soon.